server/services/FakeNameGeneratorService.ts

```diff
@@ -0,0 +1,11 @@
+export default class FakeNameGeneratorService {
+    static NAMES = [
+        "Charles",
+        "Xavier",
+        "Nicolas"
+    ];
```

Comment:
We could put **our** names instead lol.

Reply:
Yeah I thought about that, we just might!